.github/workflows/verify.yaml

```diff
@@ -197,3 +193,194 @@ jobs:
+
+      - name: AWS Tests
+        uses: ./.github/actions/run-tests
+        with:
+          command: ./gradlew -p extensions/aws test -DincludeTags="AwsS3IntegrationTest"
+
+  Daps-Integration-Tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: reset permissions to permit checkout (because the omejdn volumes)
+        run: sudo chown -R $USER:$USER ${{ github.workspace }}
+
+      - uses: actions/checkout@v2
+
+      - run: docker run -d --rm -p 4567:4567 -v ${{ github.workspace }}/extensions/iam/daps/src/test/resources/config:/opt/config -v ${{ github.workspace }}/extensions/iam/daps/src/test/resources/keys:/opt/keys ghcr.io/fraunhofer-aisec/omejdn-server:1.3.1
+
+      - uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - name: Daps Integration Tests
+        uses: ./.github/actions/run-tests
+        with:
+          command: ./gradlew -p extensions/iam/daps test -DincludeTags="DapsIntegrationTest"
+
+  End-To-End-Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - name: End to End Integration Tests
+        uses: ./.github/actions/run-tests
+        with:
+          command: ./gradlew test -DincludeTags="EndToEndTest"
+
+      - name: "Publish Gatling report"
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: Gatling reports
+          path: '**/build/reports/gatling/**'
+
+  Component-Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - name: Component Tests
+        uses: ./.github/actions/run-tests
+        with:
+          command: ./gradlew test -DincludeTags="ComponentTest"
+
+  Micrometer-Integration-Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - name: Download opentelemetry javaagent.
+        run: wget -q https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.0/opentelemetry-javaagent.jar
+      - name: Micrometer Integration Tests
+        uses: ./.github/actions/run-tests
+        with:
+          command: ./gradlew -p core/micrometer test -DincludeTags="OpenTelemetryIntegrationTest"
+
+  Upload-Test-Report:
+    needs:
+      - Unit-Tests
+      - Azure-Storage-Integration-Tests
+      - Azure-CosmosDB-Integration-Tests
+      - Aws-Integration-Tests
+      - Daps-Integration-Tests
+      - End-To-End-Tests
+      - Component-Tests
+      - Micrometer-Integration-Tests
```

Comment:
nit: just out of curiosity, is there a way to say run this job as a last job that would not need us to keep the `needs` section up to date as we add more test jobs in future?

(Just a question nothing needs to be changed here)

Reply:
not that I know